Makefile

```diff
@@ -584,10 +615,17 @@ docopen: $(apidocs_html)
 docclean:
 	$(RM) -r out/doc
 
+# Prepare the build for running the tests
+# Note that this is not run by the CI jobs on ARM or Linux with FIPS
```

Comment:
now that I've seen this and noticed `CONFIG_FLAGS` I've gone and edited node-test-commit-linux-fips to not be an exception here, it now does
```
PYTHON=python FLAKY_TESTS=$FLAKY_TESTS_MODE CONFIG_FLAGS=--openssl-fips=$WORKSPACE/fipscanisterdir $MAKE run-ci -j $JOBS
```

so it's using `run-ci` like the rest. I'll do the same for the new jobs I'm building out in node-test-commit-linux-linked that do a similar `./configure` override thing.

So the only thing I'd suggest about this option is that you note that it's called by `run-ci`, we don't actually call it directly anywhere.

_(FYI I also think that node-test-commit-linux-fips wasn't running tests in parallel due to a mis-application of `JOBS` and `-j` and I think that should be fixed now, that would explain why it's been taking more time than it should in some instances).